examples/cross_val/scripts/cross_val_algo_random_forest.py

```diff
@@ -65,5 +66,5 @@
 
 print(f'Tuples keys have been saved to {os.path.abspath(folds_keys_path)}')
 print('\nRun the following commands to track the status of the tuples:')
-print(f'    substra list traintuple -f "traintuple:tag:{tag}" --profile node-1')
-print(f'    substra list testtuple -f "testtuple:tag:{tag}" --profile node-1')
+print(f'    substra list traintuple -f "traintuple:tag:{tag}"')
```

Comment:
I think the profile is necessary here, otherwise calls will use the default profile and fail if the default profile isn't the same as node-1.

Reply:
The 'default' and 'node-1' profiles are actually equal, one is used to login with the CLI and the other with the SDK, so there is no token conflict.